node/cn/api_tracer.go

```diff
@@ -768,10 +768,10 @@ func (api *PrivateDebugAPI) traceTx(ctx context.Context, message blockchain.Mess
 		defer cancel()
 
 	case config == nil:
-		tracer = vm.NewStructLogger(nil)
+		tracer = vm.NewStructLogger(config.LogConfig)
```

Comment:
Doesn't this trigger nil pointer error?

Reply:
Oh, I will change it back to the original code. Thanks!